packages/lib/file-api-driver-webdav.js

```diff
@@ -224,6 +226,13 @@ class FileApiDriverWebDav {
 		await this.delete('');
 		await this.mkdir('');
 	}
+
+	initialize() {
+		const providerCheck = checkProviderIsUnsupported(Setting.value('sync.6.path'));
+		if (providerCheck.isUnsupported === true) {
+			throw new Error(`The WebDAV implementation of ${providerCheck.unsupportedProvider} is incompatible with Joplin, and as such is no longer supported. Please use a different sync method.`);
```

Comment:
Please make sure it can be translated (`_('.... %s ...', providerName)`)

Reply:
Done.